libmscore/stem.cpp

```diff
@@ -112,7 +112,7 @@ void Stem::layout()
                   }
             }
 
-      qreal lw5 = _lineWidth * .5 * mag();
+      qreal lw5 = 0.5 * lineWidthMag();
```

Comment:
`.5` is good enough actually.

Reply:
I know, but for some reason during my career I never liked numbers starting with a decimal point :-(

Reply:
MuseScore doesn't seem to have a coding principle for that, so I guess both is OK?

Reply:
Not I'm aware of. Could'n find it anyway.

Reply:
Either is used frequently in the existing code base